activerecord/test/cases/attribute_methods_test.rb

```diff
@@ -1135,6 +1135,34 @@ def name
     assert_equal "New topic", topic.title_alias_to_be_undefined
   end
 
+  # Test for de765e37c1d
```

Comment:
```suggestion
```

If you want add this information in the commit message, that shoul de sufficient to know why this test was added.

Reply:
Sure, will remove.